Public/Src/Engine/Scheduler/Tracing/Log.cs

```diff
@@ -319,7 +319,7 @@ internal static string PipWriteFileFailedMessage(string pipDescription, string p
             Keywords = (int)Events.Keywords.UserMessage,
             EventTask = (ushort)Events.Tasks.Scheduler,
             Message = "[{pipDescription}] Failed to materialize pip dependencies content from cache: {errorMessage}")]
-        internal abstract void PipMaterializeDependenciesFromCacheFailure(LoggingContext loggingContext, string pipDescription, string errorMessage);
+        public abstract void PipMaterializeDependenciesFromCacheFailure(LoggingContext loggingContext, string pipDescription, string errorMessage);

```

Comment:
>public [](start = 8, length = 6)

looks like it's only needed for the test, right? can we achive the same effect by adding 'internalsVisibleTo : [ name of test namespace/dsc ]' ?